From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-shawja). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

MSSedusch committed:
add SUSE ha guide

MSSedusch committed:
add overview image

MSSedusch committed:
fix format

MSSedusch committed:
add information about keep alive and sapinst parameter

MSSedusch committed:
test new format

MSSedusch committed:
add sap notes and other guides

MSSedusch committed:
add notes and guides

MSSedusch committed:
test new format

MSSedusch committed:
test new format

MSSedusch committed:
test format

MSSedusch committed:
test format

MSSedusch committed:
add drbd devices for ASCS and ERS

dominicbetts committed:
Revert "Temporarily remove jobs article"

This reverts commit 5629c71e704444f8d79bd95fdaa5f9c46cf4b3d9.

dominicbetts committed:
Add missing image

MSSedusch committed:
update article to setup two clusters

MSSedusch committed:
remove internal network todo

RicksterCDN committed:
first pass at ASM doc

MSSedusch committed:
fix merge conflict

RicksterCDN committed:
up to preparing client

RicksterCDN committed:
up to preparing client again

Juliako committed:
Update media-services-get-media-processor.md

Juliako committed:
Update media-services-rest-get-media-processor.md

Juliako committed:
Update media-services-get-media-processor.md

Juliako committed:
Update media-services-rest-get-media-processor.md

MSSedusch committed:
add SWPM patch level

swkrish committed:
Merge pull request #7 from MicrosoftDocs/master

updating private branch

ShilpiGautam committed:
Update billing-enterprise-api-usage-detail.md

Added data dictionary. The description for store service identifier is still empty. I am awaiting comments on this.

ShilpiGautam committed:
test commit

ShilpiGautam committed:
Updating documentation for ACI GA

ShilpiGautam committed:
Updating documentation for ACI GA

ShilpiGautam committed:
Updating documentation for ACI GA

dominicbetts committed:
Add job query

dominicbetts committed:
Fix merge conflict

Commit:
IPv6 updates.

ShilpiGautam committed:
Updating documentation for ACI GA

ShilpiGautam committed:
Updating documentation for ACI GA

ShilpiGautam committed:
Updating documentation for ACI GA

ShilpiGautam committed:
Updating documentation for ACI GA

ShilpiGautam committed:
Updating documentation for ACI GA

swkrish committed:
Added upgrade article

ShilpiGautam committed:
Updating documentation for ACI GA based on pre-review feedback

ShilpiGautam committed:
Updating documentation for ACI GA based on pre-review feedback

anosov1960 committed:
Update sql-database-geo-replication-overview.md

anosov1960 committed:
Update sql-database-geo-replication-overview.md

anosov1960 committed:
Update sql-database-geo-replication-overview.md

anosov1960 committed:
Update sql-database-geo-replication-overview.md

anosov1960 committed:
Update sql-database-geo-replication-overview.md

swkrish committed:
added more material

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-upgrade-preview-authentication-agents.md

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-upgrade-preview-authentication-agents.md

swkrish committed:
Add files via upload

swkrish committed:
added screenshot

swkrish committed:
added screenshot of Control Panel

swkrish committed:
added border

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-upgrade-preview-authentication-agents.md

ShilpiGautam committed:
Updating documentation for ACI GA based on pre-review feedback

CarlRabeler committed:
Update sql-database-security-tutorial.md

swkrish committed:
Added PTA Azure Portal screenshots

swkrish committed:
Update TOC.md

swkrish committed:
added Azure portal screenshots

swkrish committed:
Add files via upload

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-upgrade-preview-authentication-agents.md

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-quick-start.md

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication.md

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-quick-start.md

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-how-it-works.md

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-smart-lockout.md

swkrish committed:
Acrolinx improvements

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-upgrade-preview-authentication-agents.md

swkrish committed:
fixed broken link

swkrish committed:
Active and inactive PTA agents - screenshot

swkrish committed:
Update active-directory-aadconnect-troubleshoot-pass-through-authentication.md

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-quick-start.md

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-quick-start.md

swkrish committed:
Add files via upload

RicksterCDN committed:
updates

RicksterCDN committed:
more updates

RicksterCDN committed:
last typo fix

dominicbetts committed:
Add TOC entry

MSSedusch committed:
fix captions

MSSedusch committed:
add toc changes

dominicbetts committed:
Add missing ToC entry

dominicbetts committed:
Add missing iclude

CarlRabeler committed:
Update sql-database-security-tutorial.md

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into forced

anosov1960 committed:
Update sql-database-geo-replication-overview.md

missed escaping <>

carolinacmoravia committed:
C54502: Unlocalized content due to incorrect code format

mimig1 committed:
Cosmos DB: Added link to ref pages and .NET SDK compat.

mimig1 committed:
fixed link text

kgremban committed:
Updated reference to header-based sign-on

kgremban committed:
Update text in Working with different on-prem and cloud section

Commit:
Merge pull request #18323 from kgremban/patch-2

Update text in Working with different on-prem and cloud section

Commit:
Merge pull request #18322 from kgremban/patch-1

Updated reference to header-based sign-on

RicksterCDN committed:
acrolinx updates

Commit:
update server name, edit

markgalioto committed:
updating the Upgrade Backup vault article with public preview information

cherylmc committed:
update

Commit:
update server name, edit

tfitzmac committed:
revised first template

cjgronlund committed:
link text

Commit:
Merge pull request #18327 from cherylmc/format4

update

swkrish committed:
link to migration guide

Commit:
Merge pull request #18325 from stevestein/SqlDb728

update server name, edit

Commit:
Merge pull request #18329 from cjgronlund/fresh-july

link text

swkrish committed:
updated paths for rebranded agent

swkrish committed:
side by side is now available

swkrish committed:
wrong location

tfitzmac committed:
formatting changes

markgalioto committed:
working on the Acrolinx issues

markgalioto committed:
one more tweak to the metadata

Lisaco88 committed:
Merge pull request #18328 from tfitzmac/0727firsttemplate

revised first template

markgalioto committed:
fixing formatting of a list

MicrosoftGuyJFlo committed:
Logo fix for ThirdLight tutorial from Vendor

RicksterCDN committed:
fixing metadata

MicrosoftGuyJFlo committed:
Logo fix for ThirdLight tutorial from Vendor

MGoedtel committed:
Updated article

Commit:
Merge pull request #18320 from mimig1/patch-123

Cosmos DB: Added link to ref pages and .NET SDK compat.

sideye committed:
Make image open in new tab

sideye committed:
Make button open in new tab

sideye committed:
Change lower button to open in new link

sideye committed:
Change bullets to numbers for overview of simulator section

sideye committed:
Add note for iothubowner string

sideye committed:
Update note

sideye committed:
Update iot-hub-get-started-create-hub-and-device.md

sideye committed:
Fix hyperlink string url

Commit:
Merge pull request #18336 from sideye/patch-2

Optimize Web Simulator with Validated Solutions from User Testing

tfitzmac committed:
fixing parameters

Commit:
Merge pull request #18337 from tfitzmac/0730deploy

fixing parameters

linda33wj committed:
fix broken links

Commit:
Merge pull request #18338 from linda33wj/master

Fix broken links

dominicbetts committed:
Fix ToC

MSSedusch committed:
Merge branch 'master' into nw-ha

MSSedusch committed:
fix numbered list

MSSedusch committed:
fix spelling

Commit:
freshness

Commit:
forgot to change a date

Commit:
Merge pull request #18306 from carolinacmoravia/patch-293

C54502: Unlocalized content due to incorrect code format

Commit:
Merge pull request #18347 from Blackmist/fresh2

Freshness

MattGLaBelle committed:
Merge pull request #12460 from MSSedusch/nw-ha

SAP: add SUSE ha guide

MattGLaBelle committed:
Merge pull request #16084 from dominicbetts/javaupload

Add java schedule jobs tutorial

Commit:
Merge pull request #3 from MicrosoftDocs/master

Merge from head fork

Lisaco88 committed:
Merge pull request #18107 from ShilpiGautam/patch-1

Patch 1

MattGLaBelle committed:
Merge pull request #17723 from RicksterCDN/OracleASM

first pass at ASM doc

Lisaco88 committed:
Merge pull request #18333 from MGoedtel/UpdateSUMSOL7282017

Updated article

dkamstra committed:
Fixed syntax error in sample JSON

Lisaco88 committed:
Merge pull request #18332 from MicrosoftGuyJFlo/SaaSThirdLightLogoFix

Logo fix for ThirdLight tutorial from Vendor

Commit:
Merge pull request #18355 from dkamstra/master

Fixed a syntax error in JSON example

Commit:
Merge pull request #18265 from CarlRabeler/patch-310

Update sql-database-security-tutorial.md

Lisaco88 committed:
Merge pull request #18326 from markgalioto/Az7-28

updating the Upgrade Backup vault article with public preview informa…

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Commit:
Fixes.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into ipv6b

CarlRabeler committed:
usability

priyaravi20 committed:
Update websocketprotocol.md

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into forced

Commit:
Merge pull request #18363 from CarlRabeler/20170331-qs

usability

v-alje committed:
Merge pull request #18358 from MicrosoftDocs/FromPublicRepo

From public repo

Lisaco88 committed:
Merge pull request #18269 from swkrish/master

Upgrade preview agents + Azure portal UI

v-shils committed:
Merge pull request #18297 from cherylmc/forced

Forced update

v-shils committed:
Merge pull request #18115 from jimdial/ipv6b

IPv6 updates.

Commit:
Merge pull request #18362 from priyaravi20/patch-5

Update websocketprotocol.md

v-shils committed:
Merge pull request #17833 from Juliako/patch-10

Update media-services-get-media-processor.md

v-shils committed:
Merge pull request #17834 from Juliako/patch-11

Update media-services-rest-get-media-processor.md

Commit:
Merge pull request #18240 from anosov1960/patch-2

Update sql-database-geo-replication-overview.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPrivateRepo

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
